client/app/hosting/database/dump/hosting-database-dump.controller.js

```diff
@@ -24,52 +22,37 @@ angular.module("App").controller(
         }
 
         loadDumps () {
-            this.dumpsLoading = true;
-            this.dumpsIds = null;
+            this.databaseDumps = undefined;
 
             return this.hostingDatabase
                 .getDumpIds(this.$stateParams.productId, this.$scope.bdd.name)
-                .then((dumpIds) => (this.dumpsIds = dumpIds))
-                .catch((err) => this.alerter.alertFromSWS(this.$scope.tr("hosting_tab_databases_get_error"), err, this.$scope.alerts.main))
-                .finally(() => {
-                    if (_.isEmpty(this.dumpsIds)) {
-                        this.dumpsLoading = false;
-                    }
-                });
+                .then((dumpIds) => dumpIds.map((id) => ({ id })))
+                .then((databaseDumps) => {
```

Comment:
Question : Is this useful to use double ".then" here instead of returning directly the result of map function ?

Reply:
It takes less lines

Reply:
Doesn't seem useful here

```
.then((dumpIds) => {
    this.databaseDumps = dumpIds.map((id) => ({ id });
    return this.databaseDumps
})
```